### PR TITLE
Bring antora 5 docs to live docs site

### PR DESCRIPTION
**Pull request by MitchC1999**
Ticket: DOC-2234, DOC-2236

Changes:
* Bringing staging 5 changes to live site, creating antora 5 docs
* Added workflows to update live 5 site without interfering with 6 or 7
* Fixed remaining issues in antora 5 builds
* Brought live 5 docs changes to antora branches

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: _new_content_templates/blank_releasenotes.adoc [hunk 1]
```diff
@@ -25,19 +25,19 @@ The following "list" should be added to `data/nav.yml`
...
-* <<accompanyingpremiumskinsandiconpackschanges,Accompanying Premium Skins and Icon Packs changes>>
-* <<accompanyingpremiumself-hostedserver-sidecomponentchanges,Accompanying Premium self-hosted server-side component changes>>
-* <<generalbugfixes,General bug fixes>>
-* <<securityfixes,Security fixes>>
-* <<deprecatedfeatures,Deprecated features>>
-* <<knownissues,Known issues>>
-* <<upgradingtothelatestversionoftinymce5,Upgrading to the latest version of TinyMCE 5>>
+{productname} vnumtxt was released for {enterpriseversion} and {cloudname} on +`WEEKDAY`, `MMM` `D`^`st|nd|th`^, `YYYY`+ It includes {productname} vnumtxt and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} vnumtxt, including:
+
+* xref:newfeatures[New features]
```
Comment (kemister85) [inline _new_content_templates/blank_releasenotes.adoc:30]: Should this be updated to match the same conventions?
```suggestion
* xref:new-features[New features]
```

File: _new_content_templates/blank_releasenotes.adoc [hunk 2]
```diff
@@ -25,19 +25,19 @@ The following "list" should be added to `data/nav.yml`
...
+* xref:enhancements[Enhancements]
+* xref:functionalitychanges[Functionality changes]
```
Comment (kemister85) [inline _new_content_templates/blank_releasenotes.adoc:32]: Also this one
```suggestion
* xref:functionality-changes[Functionality changes]
```

File: _new_content_templates/blank_releasenotes.adoc [hunk 3]
```diff
@@ -25,19 +25,19 @@ The following "list" should be added to `data/nav.yml`
...
+* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium Plugin changes]
+* xref:accompanyingpremiumskinsandiconpackschanges[Accompanying Premium Skins and Icon Packs changes]
+* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium self-hosted server-side component changes]
+* xref:general-bug-fixes[General bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:deprecatedfeatures[Deprecated features]
+* xref:knownissues[Known issues]
+* xref:upgrading-to-the-latest-version-of-tinymce-5[Upgrading to the latest version of TinyMCE 5]
```
Comment (kemister85) [inline _new_content_templates/blank_releasenotes.adoc:40]:
```suggestion
* xref:new-features[New features]
* xref:enhancements[Enhancements]
* xref:functionality-changes[Functionality changes]
* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium Plugin changes]
* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
* xref:accompanying-premium-self-hosted-server-side-component-changes[Accompanying Premium self-hosted server-side component changes]
* xref:general-bug-fixes[General bug fixes]
* xref:security-fixes[Security fixes]
* xref:deprecated-features[Deprecated features]
* xref:known-issues[Known issues]
* xref:upgrading-to-the-latest-version-of-tinymce-5[Upgrading to the latest version of TinyMCE 5]
```

Comment (MitchC1999) [reply _new_content_templates/blank_releasenotes.adoc:40]: Not going to worry about this, there shouldn't be any new release notes. I didn't use this template anyway when migrating the live release notes.

**Review by kemister85 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,7 +1,5 @@
 # TinyMCE documentation
 
-[![wercker status](https://app.wercker.com/status/4d4c743635332430f9d25acae1be5218/s/main "wercker status")](https://app.wercker.com/project/bykey/4d4c743635332430f9d25acae1be5218)
-
 This project maintains the documentation for TinyMCE at
 [https://www.tiny.cloud/docs](https://www.tiny.cloud/docs). If you have any
```
Comment (kemister85) [inline README.md:4]: @MitchC1999 should we update this to `https://www.tiny.cloud/docs/tinymce/latest/`?

Comment (MitchC1999) [reply README.md:4]: No. That URL will (hopefully) eventually point to a docs homepage, for all versions and products, so it is still an appropriate link. I feel like I mentioned this at some point.

Comment (kemister85) [reply README.md:4]: Now you mention it, i believe we did chat about this #noted4later.

**Comment by MitchC1999** [comment]
- [ ] Merge Commit

**Review by ltrouton — APPROVED** [review]

**Pull request merged**